pylearn2/datasets/dataset.py

```diff
@@ -2,21 +2,25 @@
 A module defining the Dataset class.
 """
 
+import abc
+import warnings
+from pylearn2.utils.iteration import resolve_iterator_class
+
 
 class Dataset(object):
 
     """
     Abstract interface for Datasets.
     """
+    #__metaclass__ = abc.ABCMeta
```

Comment:
Adding the metaclass leads to a test failure. In my opinion the test is wrong (it instantiates an empty dataset), but apparently it is not. Anyway, I will remove this line.